src/main/resources/lang/simplifiedchinese.lang

```diff
@@ -15,7 +15,7 @@ skript:
 	invalid reload: Skript只能通过Bukkit的“/reload”或Skript的“/skript reload”命令来重新加载。
 	no scripts: 没有发现任何脚本，也许你应该写一些 ;)
 	no errors: 所有脚本都已加载，没有错误。
-	scripts loaded: 已加载%s个脚本，包括%s个结构和%s个命令（%s内）
+	scripts loaded: 已加载 %s 个脚本，总共包含 %s 个结构。 (耗时 %s）
```

Comment:
```suggestion
	scripts loaded: 已加载%s个脚本，总共包含%s个结构。 (耗时%s）
```
If for the sake of consistency across the whole lang file as it is currently written with mixture of non-Chinese and Chinese without spacing between them:
Loaded 28 scripts with 28 structures (2828ms) 
- 已加载 28 个脚本，总共包含 28 个结构。（耗时 2828ms）
VS
- 已加载28个脚本，总共包含28个结构。（耗时2828ms）
(Certain punctuation marks in Chinese comes along with spacing thats out of our control)
For my case I highly prefer the latter, but at the end I wonder reformatting all in this PR is out of scope and if so then then apply the current change which removes spacing and then the lang may be reformatted in a later PR, if demands surges.

Reply:
If you want to redo the entire SimplifiedChinese PR go for it, I don't know Chinese at all and it would be helpful for someone who knew Chinese to make all of those changes 🙂

Reply:
Then just merge the change I suggested and youre good to go